relayer-cli/src/commands.rs

```diff
@@ -63,6 +65,10 @@ pub enum CliCmd {
     #[clap(subcommand)]
     Upgrade(UpgradeCmds),
 
+    /// Clear entities (e.g. packets) on chains
```

Comment:
This help text could be improved, please make your suggestions.

Reply:
One suggestion
> Clear outstanding packets (i.e., packet-recv and packet-ack) on a given channel.

Reply:
This text is for the entire `clean` top-level command.

Reply:
Good point, I didn't realize that! I would simply replace `entities` by `objects`.